test/test_type_hints.py

```diff
@@ -204,12 +204,38 @@ def is_torch_mypyini(path_to_file):
             self.skipTest("Typeannotations in numpy-1.20.0-dev are broken")
 
         cwd = os.getcwd()
+        # TODO: Would be better not to chdir here, this affects the entire
+        # process!
```

Comment:
good point adding the `try`-`except`. not sure the `TODO` is needed here, this was hard to avoid and it seems harmless - parallel test execution with `pytest-xdist` should use separate processes, and that's the one relevant case I can think of.

Reply:
It's just a general hygiene thing, from my days when I could actually run thread concurrently :) You could trigger this via just the threading module, and people do use it in Python. OTOH I agree that it is relatively harmless and I'm not going to go out of my way to try to fix it, especially if mypy doesn't support explicitly specifying what cwd to run in!

Reply:
According to mypy developers, it's not supposed to throw, but I guess it's a good hygiene